content/v2/openapi.yml

```diff
@@ -1945,6 +1946,124 @@ paths:
         '404':
           $ref: '#/components/responses/404'
       summary: Delete a contact
+  '/{account}/registrant_changes':
+    get:
+      description: List contact changes in the account.
```

Comment:
We are using two languages, contact changes to the "end user" and registrant change as domain scope. I'm unsure if it's confusing and would appreciate suggestions.